File: app/src/main/java/com/give3/gizrog/give3/items/Task.java
package com.give3.gizrog.give3.items;

import java.util.ArrayList;
import java.util.List;

public class Task {

    String title;

    List<SubTask> subtasks = new ArrayList<>();

    Task() {}

    Task(String title) {
        this.title = title;
    }

    public class SubTask {

        String subTaskTitle;

        SubTask() {}

        SubTask(String subTaskTitle) {
            this.subTaskTitle = subTaskTitle;
        }


    }

}

File: app/src/main/java/com/give3/gizrog/give3/items/Task.kt
package com.give3.gizrog.give3.items

import java.util.ArrayList

class Task {

    internal var title: String = ""

    internal var subtasks: List<SubTask> = ArrayList()

    internal constructor() {}

    internal constructor(title: String) {
        this.title = title
    }

    inner class SubTask {

        internal var subTaskTitle: String = ""

        internal constructor() {}

        internal constructor(subTaskTitle: String) {
            this.subTaskTitle = subTaskTitle
        }


    }

}
